htdocs/ticket/card.php

```diff
@@ -212,7 +212,8 @@
 			$db->begin();
 
 			$getRef = GETPOST("ref", 'alphanohtml');
-			if ($object->fetch('', $getRef) > 0) {
+			$refcheck_object = new Ticket($db);
+			if ($refcheck_object->fetch('', $getRef) > 0) {
```

Comment:
This looks strange.
Doing a fetch from $object that is a Ticket object or from $refcheck_object that is also a Ticket should have the same result,n whatever is the status of $object before the fetch.